packages/opentelemetry-core/src/platform/browser/log-level.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LogLevel } from '../../common/types';
+
+/**
+ * @returns default log level until below issue is tackled.
+ * TODO: https://github.com/open-telemetry/opentelemetry-js/issues/1096
+ */
+export function getEnvironmentalOrDefaultLogLevel(): LogLevel {
+  return LogLevel.INFO;
```

Comment:
This should be compatible with bundlers like Webpack ( that inject a process variable into the bundle ) and still work with vanilla projects.

```suggestion
  const processLogLevel = (typeof process !== 'undefined' && process.env && process.env.OTEL_LOG_LEVEL)
    || window.OTEL_LOG_LEVEL;

    return LogLevel[processLogLevel as LogLevelString]
      ||  LogLevel.INFO
}

```

Reply:
@obecny @draffensperger could you please review this suggestion by @iliasbhal ?

Reply:
I think we will end up with many code duplication. I added my suggestion here: https://github.com/open-telemetry/opentelemetry-js/pull/974#issuecomment-638420365

I also think that we should probably have enum to be used with environment variables. And if we have platform specific environment, then it will be a matter of reading the log level in one place only without worrying if this is a window, global, process.env or something else.